src/js/bootstrap-slider.js

```diff
@@ -1453,7 +1453,7 @@ const windowIsDefined = (typeof window === "object");
 				this._setDataVal(newValue);
 				this.setValue(newValue, false, true);
 
-				this._pauseEvent(ev);
+				ev.returnValue = false;
```

Comment:
what happens if we stick `this._pauseEvent(ev);` after `ev.returnValue = false;`? Does it still resolve your issue?

something like:
```js
this._pauseEvent(ev);
ev.returnValue = false;
```